mne/report.py

```diff
@@ -571,6 +571,14 @@ def _build_html_slider(slices_range, slides_klass, slider_id):
             <img alt=""
              src="data:image/png;base64,{{img}}">
         {{endif}}
+    {{elif image_format == 'gif'}}
+        {{if scale is not None}}
+            <img alt="" style="width:{{width}}%;"
+             src="data:image/gif;base64,{{img}}">
```

Comment:
you should just write 15 lines of python script first which reads a gif, converts to base64 and outputs an html. That will help you debug.